app/views/verify/address/index.html.slim

```diff
@@ -1,5 +1,14 @@
-p Pick an address verification mechanism
+h1.h2
+  = t('idv.titles.select_verification')
+p
+  = t('idv.messages.select_verification', sp_name: @sp_name)
```

Comment:
ahh ok. can I get a quick confirm from @pkarman or someone else? I know that I have gone through the LOA3 flow without an SP several times, but I am a developer so I know that my behavior is probably abnormal